modules/nf-core/gatk4/selectvariants/main.nf

```diff
@@ -9,6 +9,7 @@ process GATK4_SELECTVARIANTS {
 
     input:
     tuple val(meta), path(vcf), path(vcf_idx)
+    path (intervals)
```

Comment:
```suggestion
    tuple val(meta), path(vcf), path(vcf_idx), path (intervals)
```

Could you move the intervals to the first line? In some cases you'll want to use different BED files for different vcfs

Reply:
Allright now to fix the tests (different input) and then it's good to go :rocket:

Reply:
Yip!